html/js/tree/Node.js

```diff
@@ -4,125 +4,36 @@ GTE.TREE = (function (parentModule) {
     /**
     * Creates a new Node.
     * @class
-    * @param {Node} [parent] Parent node. If null, this is root.
+    * @param {ISet} [iSet] iSet that contains the node. Needed for sending onClicks to it
+    * @param {Number} [level] Level of the node
```

Comment:
If in the ISet.js constructor you remove the square brackets from parameter names, do it here too, for consistency.